source/tests/system/nidcpower_session_tests.cpp

```diff
@@ -83,12 +83,18 @@ class NiDCPowerSessionTest : public ::testing::Test {
   std::unique_ptr<dcpower::NiDCPower::Stub> nidcpower_stub_;
 };
 
+// InitializeWithIndependentChannels was added in dcpower 20.7.
+// To support testing with older versions: skip tests for InitializeWithIndependentChannels if
+// it returns an unsupported status.
+#define GTEST_SKIP_IF_UNSUPPORTED(status) \
+  if (status.error_code() == ::grpc::NOT_FOUND) GTEST_SKIP() << "Function not supported.";
+
 TEST_F(NiDCPowerSessionTest, InitializeSessionWithDeviceAndSessionName_CreatesDriverSession)
 {
   dcpower::InitializeWithIndependentChannelsResponse response;
-  ::grpc::Status status = call_initialize_with_independent_channels(kTestRsrc, kOptionsString, kTestSession, &response);
+  auto status = call_initialize_with_independent_channels(kTestRsrc, kOptionsString, kTestSession, &response);
 
-  EXPECT_TRUE(status.ok());
+  GTEST_SKIP_IF_UNSUPPORTED(status);
   EXPECT_EQ(0, response.status());
```

Comment:
We should still add `EXPECT_TRUE(status.ok())` to catch any other errors apart from NOT_FOUND for this and other tests below.